platform/web/display_server_web.h

```diff
@@ -106,10 +110,13 @@ class DisplayServerWeb : public DisplayServer {
 
 	static void request_quit_callback();
 	static void window_blur_callback();
-	static void update_voices_callback(int p_size, const char **p_voice);
-	static void update_clipboard_callback(const char *p_text);
+	static void _update_voices_callback(int p_size, const char **p_voice);
+	static void update_voices_callback(const Vector<String> p_voices);
```

Comment:
```suggestion
	static void update_voices_callback(const Vector<String> &p_voices);
```
I'd pass by reference here as it's already const

Reply:
Thanks!